src/convertToObject.js

```diff
@@ -5,8 +5,28 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const getDeclarationAsStrings = sourceString
+    .split(';')
+    .map((rawDeclaration) => rawDeclaration.trim());
```

Comment:
The variable name `getDeclarationAsStrings` is a bit misleading as it implies a function (due to the 'get' prefix), but it's actually an array of strings. Consider renaming it to something more indicative of its contents, like `declarations`.